packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -72,15 +88,22 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
 
   constructor(props: ICalendarProps) {
     super(props);
-    let currentDate = props.value && !isNaN(props.value.getTime()) ? props.value : (props.today || new Date());
+    let currentDate =
+      props.value && !isNaN(props.value.getTime())
```

Comment:
It looks like new formatting was applied to many areas of the files (even though you did not edit them)!